shell/browser/api/electron_api_tray.cc

```diff
@@ -93,14 +93,17 @@ gin::Handle<Tray> Tray::New(gin_helper::ErrorThrower thrower,
 void Tray::OnClicked(const gfx::Rect& bounds,
                      const gfx::Point& location,
                      int modifiers) {
+  v8::HandleScope scope(isolate());
   EmitCustomEvent("click", CreateEventFromFlags(modifiers), bounds, location);
```

Comment:
I think this is better handled in `event_emitter_mixin.h` for these event methods `Emit`, `EmitCustomEvent` etc

Reply:
I don't see how that would solve this here as `CreateEventFromFlags` returns a `v8::Local`, so needs a handle scope.